x/group/genesis.go

```diff
@@ -0,0 +1,31 @@
+package group
+
+import (
+	"github.com/cosmos/cosmos-sdk/codec/types"
+)
+
+// NewGenesisState creates a new genesis state with default values.
+func NewGenesisState() *GenesisState {
+	return &GenesisState{}
+}
+
+func (s GenesisState) Validate() error {
+	return nil
```

Comment:
could you actually validate the `GenesisState` data? ie validate groups, group accounts, etc.